04-Stacks-And-Queues/class-lesson.instructor.md

```diff
@@ -0,0 +1,70 @@
+# Lesson Plan
+
+Stacks and Queues are fairly straightforward data structures, but are included in the curriculum for completeness as they are a fundamental part of most computer science curriculums and often used in interviews.
+
+They are expanded on in the project using a circular buffer. Students often find using the `%` operator to traverse an array in a circular buffer. Demonstrating it's use in a loop can be quite helpful.
```

Comment:
It seems like the sentence below might be incomplete, I'm having some trouble understanding what issue students face with the % operator.

> Students often find using the `%` operator to traverse an array in a circular buffer.